workloads/kube-burner/run.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/bash
+
+. common.sh
+
+deploy_operator
+check_running_benchmarks
+
+case ${WORKLOAD} in
+  cluster-density)
+    WORKLOAD_TEMPLATE=workloads/cluster-density/cluster-density.yml
+    METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics-aggregated.yaml}
+    export TEST_JOB_ITERATIONS=${JOB_ITERATIONS:-1000}
+  ;;
+  node-density)
+    WORKLOAD_TEMPLATE=workloads/node-pod-density/node-pod-density.yml
+    METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics.yaml}
+    NODE_COUNT=${NODE_COUNT:-4}
+    PODS_PER_NODE=${PODS_PER_NODE:-250}
+    label_nodes regular
+  ;;
+  node-density-heavy)
+    WORKLOAD_TEMPLATE=workloads/node-density-heavy/node-density-heavy.yml
+    METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics.yaml}
+    NODE_COUNT=${NODE_COUNT:-4}
+    PODS_PER_NODE=${PODS_PER_NODE:-250}
+    label_nodes heavy
+  ;;
+  pod-density)
+    WORKLOAD_TEMPLATE=workloads/node-pod-density/node-pod-density.yml
+    METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics.yaml}
+    export TEST_JOB_ITERATIONS=${PODS:-1000}
+  ;;
+  pod-density-heavy)
+    WORKLOAD_TEMPLATE=workloads/node-density-heavy/node-density-heavy.yml
+    METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics.yaml}
+    export TEST_JOB_ITERATIONS=${PODS:-1000}
+  ;;
+  max-namespaces)
+    WORKLOAD_TEMPLATE=workloads/max-namespaces/max-namespaces.yml
+    METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics-aggregated.yaml}
+    export TEST_JOB_ITERATIONS=${NAMESPACE_COUNT:-1000}
+  ;;
+  max-services)
+    WORKLOAD_TEMPLATE=workloads/max-services/max-services.yml
+    METRICS_PROFILE=${METRICS_PROFILE:-metrics-profiles/metrics-aggregated.yaml}
+    export TEST_JOB_ITERATIONS=${SERVICE_COUNT:-1000}
+  ;;
+  custom)
+  ;;
+  *)
+     log "Unkonwn workload ${WORKLOAD}, exiting"
+     exit 1
+  ;;
+esac
+
+log "###############################################"
+log "Workload: ${WORKLOAD}"
```

Comment:
Should we also specify the workload template and alert template if they are provided/different than the norm?

Reply:
Well, alert template could make sense. Workload template is deduced from the workload :)

Reply:
Adding alert template though

Reply:
Right. My thinking was if it was a custom workload they would supply a template file which may be useful to denote in the output